data/installation_environments.yml

```diff
@@ -33,6 +33,9 @@ linux:
               - postgresql
               - postgresql-libs
               - redis
+            development:
+              - libidn
+        sudo: false
```

Comment:
Okay, sorry for bikeshedding, but `sudo` alone is probably not clear enough (what does that mean? does the system offer sudo, do the install guides need sudo, ...?). How about `needs_sudo`?

Reply:
Makes sense, but since it's more about if sudo is available by default (to prevent sudo command not found errors), I called it `has_sudo` now.

Reply:
Perfect!